boards/arm64/mimx93_evk/mimx93_evk_a55.dts

```diff
@@ -30,6 +30,14 @@
 	};
 };
 
+&lpuart1 {
+	status = "okay";
```

Comment:
Don't enable lpuart1 by default, you can enable it in project's overlay dts if you need to use it.

Reply:
done.